nixos/modules/services/mail/postsrsd.nix

```diff
@@ -200,6 +200,14 @@ in
         '';
       };
 
+      configurePostfix = lib.mkOption {
```

Comment:
Should we assert that postfix is enabled?

Reply:
The resulting configuration is already guarded around `config.services.postfix.enable`. This is like it is in tlsrpt and postfix-tlspol.